File: fizzbuzz.py
string = ""

for x in range(1,101):
	if x % 3 == 0:
		string += 'fizz'
	if x % 5 == 0:
		string += 'buzz'
	if x % 3 != 0 and x % 5 != 0:
		string += str(x)
	string += " "

print(string)

File: fizzbuzz.js
var string = "";

for (var i = 1; i < 101; i++) {
	if (i % 3 === 0) {
		string += 'fizz';
	}
	if (i % 5 === 0) {
		string += 'buzz';
	}
	if (i % 3 && i % 5){
		string += i.toString();
	}
	string += " ";
}

string;